package.json

```diff
@@ -5,8 +5,12 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
-    "run_etl_local": "npm run build && node --env-file=.env-local-etl ./public/dist/src/etl/index.js",
-    "run_etl_production": "npm run build && node --env-file=.env-production-etl ./public/dist/src/etl/index.js",
+    "run_arbo_etl_local": "npm run build && node --env-file=.env-local-etl ./public/dist/src/etl/arbo/index.js",
+    "run_arbo_etl_production": "npm run build && node --env-file=.env-production-etl ./public/dist/src/etl/arbo/index.js",
+    "run_arbo_etl_production_no_build": "node --env-file=.env-production-etl ./public/dist/src/etl/arbo/index.js",
+    "run_team_etl_local": "npm run build && node --env-file=.env-local-etl ./public/dist/src/etl/team/index.js",
+    "run_team_etl_production": "npm run build && node --env-file=.env-production-etl ./public/dist/src/etl/team/index.js",
+    "run_team_etl_production_no_build": "node --env-file=.env-production-etl ./public/dist/src/etl/team/index.js",
```

Comment:
Moved a bunch of stuff around to make an arbo ETL and an employee ETL. The arbo ETL still works hahaha